prime-router/src/main/kotlin/common/HttpClientUtils.kt

```diff
@@ -312,7 +327,7 @@ class HttpClientUtils {
             httpClient: HttpClient? = null,
```

Comment:
The suggested patch removes the ability to pass in a custom client, so this deviates from that. I think not applying that change is probably fine, seems more flexible to give clients the option of providing their own client for now. I can't think of any downsides at the moment.

Reply:
ack